tests/desktop/test_login_logout.py

```diff
@@ -86,7 +91,7 @@ def test_logout_from_translate_kb_article_page(self, base_url, selenium, variabl
         kb_translate_pg.click_translate_language('Deutsch (de)')
 
         # sign out
-        register_page = kb_translate_pg.sign_out()
-        register_page._page_title = 'Anmelden / Registrieren | Mozilla-Hilfe'
-        register_page.is_the_current_page
-        assert register_page.header.is_user_logged_out
+        login_page = kb_translate_pg.sign_out()
+        login_page.url_kwargs['locale'] = 'de'
```

Comment:
@bobsilverberg this is also interesting. The previous action resulted in the locale changing. As these keyword arguments are not currently passed between page objects (they easily could be) it was necessary to change the locale value here.